SPECS/core-packages/core-packages.spec

```diff
@@ -84,6 +84,9 @@ Requires:   vim
 Requires:   wget
 Requires:   xz
 Requires:   zlib
+Requires:   gawk
```

Comment:
Do these packages need to be added to every image, or just the CBL-Mariner Core image? This meta package is used by almost all image configurations, even containers. 


If these packages are targetting just CBL-Mariner Core then the relevant package lists / configs under `toolkit/imageconfigs` should be updated instead.

Reply:
+1. Why cannot we modify the spec for Azsec Pack instead? That sounds like a more appropriate place for dependencies.

Reply:
The previous PR title was misleading. Azsec Pack won't be directly enabled in Mariner Core at the moment, but it's needed in images like AFD, AKS, PilotFish etc. But we want to verify azsec pack in Mariner Core to make sure that it works in all derivatives and thus I'm adding the required dependencies here.